features/cells/src/main/java/com/wire/android/feature/cells/ui/versioning/VersionHistoryScreen.kt

```diff
@@ -0,0 +1,143 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.feature.cells.ui.versioning
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.feature.cells.R
+import com.wire.android.feature.cells.ui.common.LoadingScreen
+import com.wire.android.navigation.WireNavigator
+import com.wire.android.navigation.annotation.features.cells.WireDestination
+import com.wire.android.navigation.style.PopUpNavigationAnimation
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.divider.WireDivider
+import com.wire.android.ui.common.preview.MultipleThemePreviews
+import com.wire.android.ui.common.scaffold.WireScaffold
+import com.wire.android.ui.common.topappbar.NavigationIconType
+import com.wire.android.ui.common.topappbar.WireCenterAlignedTopAppBar
+import com.wire.android.ui.theme.WireTheme
+
+@WireDestination(
+    style = PopUpNavigationAnimation::class,
+    navArgsDelegate = VersionHistoryNavArgs::class,
+)
+@Composable
+fun VersionHistoryScreen(
+    navigator: WireNavigator,
+    modifier: Modifier = Modifier,
+    versionHistoryViewModel: VersionHistoryViewModel = hiltViewModel()
+) {
+    VersionHistoryScreenContent(
+        versionsGroupedByTime = versionHistoryViewModel.versionsGroupedByTime.value,
+        modifier = modifier,
+        isFetchingContent = versionHistoryViewModel.isFetchingContent.value,
+        navigateBack = { navigator.navigateBack() }
+    )
+}
+
+@Composable
+private fun VersionHistoryScreenContent(
+    versionsGroupedByTime: List<VersionGroup>,
+    modifier: Modifier = Modifier,
+    isFetchingContent: Boolean,
+    navigateBack: () -> Unit = {}
+) {
+    WireScaffold(
+        modifier = modifier,
+        topBar = {
+            WireCenterAlignedTopAppBar(
+                onNavigationPressed = navigateBack,
+                title = stringResource(R.string.version_history_top_appbar_title),
+                navigationIconType = NavigationIconType.Close(),
+                elevation = dimensions().spacing0x,
+            )
+        },
+    ) { innerPadding ->
+
+        AnimatedVisibility(
+            visible = isFetchingContent,
+            enter = fadeIn(),
+            exit = fadeOut()
+        ) { LoadingScreen() }
+
+        AnimatedVisibility(
+            visible = !isFetchingContent,
+            enter = fadeIn(),
+            exit = fadeOut()
+        ) {
```

Comment:
Instead of using two `AnimatedVisibility`, you can use just one `AnimatedContent` or `Crossfade` to animate between multiple states